storage-proofs/src/vde.rs

```diff
@@ -141,9 +141,11 @@ pub fn create_key_from_tree<H: Hasher>(
     hasher.update(id.as_ref());
 
     // The hash is about the parents, hence skip if a node doesn't have any parents
+    let mut scratch: [u8; NODE_SIZE] = [0; NODE_SIZE];
```

Comment:
Is this the correct size?

Reply:
I believe so, thought allocation should happen inside of the `if` statement